staging/src/k8s.io/apimachinery/pkg/util/net/interface.go

```diff
@@ -59,19 +60,22 @@ func getRoutes(input io.Reader) ([]Route, error) {
 			continue
 		}
 		fields := strings.Fields(line)
-		routes = append(routes, Route{})
-		route := &routes[len(routes)-1]
-		route.Interface = fields[0]
-		ip, err := parseIP(fields[1])
+		dest, err := parseIP(fields[1])
 		if err != nil {
 			return nil, err
 		}
-		route.Destination = ip
-		ip, err = parseIP(fields[2])
+		gw, err := parseIP(fields[2])
```

Comment:
Actually, this needs to stay as parseIP(), as it is used in part 3 for both IPv4 and IPv6 processing. I'm reverting it in part 3 (46874).